src/lib/xiaomi.ts

```diff
@@ -339,11 +351,17 @@ export const numericAttributes2Payload = async (msg: Fz.Message, meta: Fz.Meta,
                 } else {
                     payload.motion_sensitivity = getFromLookup(value, {1: 'low', 2: 'medium', 3: 'high'});
                 }
+            } else if (['ZNXDD01LM'].includes(model.model)) {
+                payload.color_temp = value;
             }
             break;
         case '103':
             if (['RTCZCGQ11LM'].includes(model.model)) {
                 payload.monitoring_mode = getFromLookup(value, {0: 'undirected', 1: 'left_right'});
+            } else if (['ZNXDD01LM'].includes(model.model)) {
+                assertNumber(value);
```

Comment:
If we don't do anything with it, why do an assertion here? It can only give troubles.

Reply:
fixed